docs/source/en/model_doc/llama2.md

```diff
@@ -55,6 +55,31 @@ come in several checkpoints they each contain a part of each weight of the model
 
 This model was contributed by [Arthur Zucker](https://huggingface.co/ArthurZ) with contributions from [Lysandre Debut](https://huggingface.co/lysandre). The code of the implementation in Hugging Face is based on GPT-NeoX [here](https://github.com/EleutherAI/gpt-neox). The original code of the authors can be found [here](https://github.com/facebookresearch/llama).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with LLaMA2. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+- A blog on how to [understand Llama 2's features and its collaboration with Hugging Face](https://huggingface.co/blog/llama2).
+- A blog on [an introduction to Llama 2](https://www.philschmid.de/llama-2) by Phil Schmid.
+
+<PipelineTag pipeline="text-generation"/>
+
+- A notebook on how to [fine-tune Llama 2 in Google Colab using QLoRA and 4-bit precision](https://colab.research.google.com/drive/1PEQyJO1-f6j0S_XJ8DV50NkpzasXkrzd?usp=sharing). 🌎
+- A notebook on how to [fine-tune the "Llama-v2-7b-guanaco" model with 4-bit qlora and generate Q&A datasets from PDFs](https://colab.research.google.com/drive/134o_cXcMe_lsvl15ZE_4Y75Kstepsntu?usp=sharing). 🌎
+
+⚗️ Optimization
+- A blog on how to [fine-tune Llama 2 using the Direct Preference Optimization (DPO) method](https://huggingface.co/blog/dpo-trl).
+- A blog on how to [instruction-tune Llama 2 for optimized performance](https://www.philschmid.de/instruction-tune-llama-2) by Phil Schmid.
+- A notebook on how to [fine-tune the Llama 2 model on a personal computer using QLoRa and TRL](https://colab.research.google.com/drive/1SYpgFpcmtIUzdE7pxqknrM4ArCASfkFQ?usp=sharing). 🌎
```

Comment:
Put the link emphasis on "notebook" rather than the title of it (same comment for the rest of the notebooks)